specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/stable/2021-11-12/hanaonazure.json

```diff
@@ -0,0 +1,707 @@
+        "x-ms-examples": {
+          "Update Tags field of a HANA instance": {
+            "$ref": "./examples/HanaInstances_PatchTags.json"
+          },
+          "Delete Tags field of a HANA instance": {
+            "$ref": "./examples/HanaInstances_PatchTags_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HanaInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HanaInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "The resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "HanaInstancesListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HanaInstance"
+          },
+          "description": "The list of SAP HANA on Azure instances."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of HANA instances."
+        }
+      },
+      "description": "The response from the List HANA Instances operation."
+    },
+    "HanaInstance": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HanaInstanceProperties",
+          "description": "HANA instance properties"
+        },
+        "systemData": {
+          "$ref": "#/definitions/systemData",
```

Comment:
There is no "systemData" defined in this json, you might want to point ref to "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"